packages/helpers/package.json

```diff
@@ -6,26 +6,29 @@
   "homepage": "https://github.com/nervosnetwork/lumos#readme",
   "license": "MIT",
   "main": "lib/index.js",
-  "types": "index.d.ts",
+  "types": "lib/index.d.ts",
```

Comment:
If this is a TypeScript package, maybe this part is not necessary?

Reply:
This package can reference from `js` and `ts`, and source typescript files will compile to `*.js` and `*.d.ts` files when publish, and exports them. It still use `js` and `.d.ts` pairs.